DIVD-2022-00060 Bitbucket v2 (#527)
DIVD-NL/web-csirt: _cases/2022/DIVD-2022-00060.md

---
layout: case
title: Command Injection vulnerability in Bitbucket Server and Data Center
author: Max van der Horst
lead: Melvin Boers
status: Open
excerpt: DIVD is scanning for parties vulnerable to CVE-2022-43781
researchers:
- Ralph Horn
- Kaj Koole
- Stan Plasmeijer
- Max van der Horst
cves:
- CVE-2022-43781
product: Bitbucket Server, Bitbucket Data Center

> versions: v7.0 to v7.5, v7.6.0 to v7.6.18, v7.7 to v7.16, v7.17.0 to v7.17.11, v7.18 to v7.20, v7.21.0 to v7.21.5 and if `mesh.enabled` is set to `false` in `bitbucket.properties`, v8.0 to v8.4.1. 

Reviewer: Best wel ingewikkelde voorwaarden, is het een idee hier in het artikel nog aandacht aan te besteden?

> recommendation: "Upgrade your affected installations to one of the by Atlassian listed fixed versions in the Security Advisory."

Reviewer: ```suggestion
recommendation: "Upgrade your affected installations to one of the fixed versions listed by Atlassian in their Security Advisory."
```

workaround: "If upgrading is not an option, a temporary mitigation step is to disable the Public Signup option. This changes the attack vector from unauthenticated to authenticated."
start: 2022-11-17
end:

> timeline:

Reviewer: Ik mis wanneer de kwetsbaarheid isnontdekt in de timeline

- start: 2022-11-17
  end:
  event: "DIVD starts tracking this vulnerability"
- start: 2022-11-22
  end:

>   event: "DIVD creates a fingerprint for this vulnerability"

Reviewer: ```suggestion
  event: "DIVD creates a method to fingerprint servers for this vulnerability"
```

- start: 2022-11-26
  end:
  event: "DIVD starts a first scan to find vulnerable parties"
- start: 2022-11-26
  end:
  event: "First version of this case file"

---

## Summary

> There is a command injection vulnerability in Bitbucket Server and Data Center that leverages environment variables to gain code execution and execute code on the system. Atlassian Cloud sites are not affected by this vulnerability. There are multiple affected versions, and in the [Security Advisory](https://confluence.atlassian.com/bitbucketserver/bitbucket-server-and-data-center-security-advisory-2022-11-16-1180141667.html), Atlassian recommends to upgrade to one of the patched versions.

Reviewer: ```suggestion
There is a command injection vulnerability in Bitbucket Server and Data Center that leverages environment variables execute code on the system. Atlassian Cloud sites are not affected by this vulnerability. There are multiple affected versions, and in their [Security Advisory](https://confluence.atlassian.com/bitbucketserver/bitbucket-server-and-data-center-security-advisory-2022-11-16-1180141667.html), Atlassian recommends to upgrade to one of the patched versions.
```

Reviewer: Is het `code execution` of `command injection`?
